python/mxnet/symbol/numpy/_symbol.py

```diff
@@ -1555,4 +1556,112 @@ def sqrt(x, out=None, **kwargs):
     return _unary_func_helper(x, _npi.sqrt, _np.sqrt, out=out, **kwargs)
 
 
+@set_module('mxnet.symbol.numpy')
+def floor(x, out=None, **kwargs):
+    r"""
+    Return the floor of the input, element-wise.
+    The floor of the scalar `x` is the largest integer `i`, such that
+    `i <= x`.  It is often denoted as :math:`\lfloor x \rfloor`.
+
+    Parameters
+    ----------
+    x : _Symbol
+        Input data.
+        Does not support scalar.
+    out : _Symbol or None, optional
+    where : array_like, optional
```

Comment:
If it's totally not supported then do not put it in the `Parameters` section. Note this down in the notes section.

Reply:
fixed and moved to notes.